src/main/java/com/cinema/controllers/MovieSessionController.java

```diff
@@ -33,7 +34,7 @@ public MovieSessionController(MovieSessionService movieSessionService, ModelMapp
     }
 
     @PostMapping
-    public void addMovieSession(@RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+    public void addMovieSession(@RequestBody @Valid MovieSessionRequestDto movieSessionRequestDto) {
```

Comment:
@NotNull doesn't work without @Valid ?

Reply:
@TimoxaV  check it)

Reply:
Yes, valid is necessary.

Reply:
Yes, we need to add Valid annotation to make our validation constraints work.